src/Relation/Nullary/Irrelevant.agda

```diff
@@ -0,0 +1,14 @@
+------------------------------------------------------------------------
+-- The Agda standard library
+--
+-- Nullary irrelevance
+------------------------------------------------------------------------
+
+{-# OPTIONS --without-K --safe #-}
+
+module Relation.Nullary.Irrelevant where
+
+open import Relation.Binary.PropositionalEquality
+
+Irrelevant : ∀ {p} → Set p → Set p
+Irrelevant P = ∀ (p₁ p₂ : P) → p₁ ≡ p₂
```

Comment:
I'm not sure this should be in its own module.
This is very reminiscent of the type of definitions we have in [`Relation.Binary.Core`](https://agda.github.io/agda-stdlib/Relation.Binary.Core.html)

Reply:
Yes, this should go in `Relation.Nullary`.

Reply:
no I can't. otherwise there is a cyclic dependency between PropositionalEquality and Nullary.

Reply:
You could import `Agda.Builtin.Equality` directly if you're only after the `_≡_` symbol.